### PR TITLE
#6 Cleanup Admin Languages and Lancheck

### PR DESCRIPTION
**Pull request by MikeyGMT**
#6 Cleanup Admin Languages and Lancheck

### CODE REVIEW

File: e107_admin/lancheck.php [hunk 1]
```diff
@@ -1847,12 +1847,12 @@ function edit_lanfiles($dir1,$dir2,$f1,$f2,$lan)
 			$rowamount = round(strlen($trans['orig'][$sk])/34)+1;
 			$hglt1=""; $hglt2="";
 			if ($trans['tran'][$sk] == "" && $trans['orig'][$sk]!="") {
-				$hglt1="<span class='label label-danger label-important e-tip' title='Missing' >";
+				$hglt1="<span class='label label-danger label-important e-tip' title=".LAN_MISSING.">";//Missing
```
Comment (CaMer0n) [inline e107_admin/lancheck.php]: It appears title is missing the single quotes around LAN_MISSING. ie. `title='".LAN_MISSING.'" `

File: e107_admin/lancheck.php [hunk 2]
```diff
@@ -1847,12 +1847,12 @@ function edit_lanfiles($dir1,$dir2,$f1,$f2,$lan)
...
 				$hglt2="</span>";
 			}
 			elseif($trans['tran'][$sk] == $trans['orig'][$sk])
 			{
-				$hglt1="<span class='label label-warning e-tip' title='Identical' >";
+				$hglt1="<span class='label label-warning e-tip' title=".LAN_CHECK_28." >";//Identical
```
Comment (CaMer0n) [inline e107_admin/lancheck.php]: Same issue as above.

**Comment by MikeyGMT** [comment]
Thanks for spotting this - now corrected.

**Comment by yesszus** [comment]
bb codes not work:
fc71008951a00fbe9b72f05f5ecc927ba2510d62

Changed the codes for:
0add4b96ec5283a03887c55d1728bb2e8abda24f

**Pull request merged**